parm/config/gfs/config.resources

```diff
@@ -565,8 +565,8 @@ case ${step} in
     ntasks=16
     case ${OCNRES} in
       "025")
-        ntasks=40
-        memory="256GB"
+        ntasks=256
+        memory="500GB"
```

Comment:
These are quite a big jump!
Is this a machine issue?  If so, perhaps this should be considered in the `config.resources.$MACHINE`

Reply:
I think this is actually because of the number of ensemble members versus a machine issue.  Gaea did require even more.